kafka-client/src/main/java/dev/responsive/kafka/internal/db/CassandraFactTable.java

```diff
@@ -230,7 +230,7 @@ public BoundStatement setOffset(
 
   @Override
   public long approximateNumEntries(final int kafkaPartition) {
-    return client.count(name(), kafkaPartition);
+    throw new UnsupportedOperationException("all is not supported on fact tables");
```

Comment:
was this intentional? this is also not `all`

Reply:
copy-pasta 😨